src/newusers.c

```diff
@@ -610,9 +610,9 @@ static void process_flags (int argc, char **argv)
 #endif
 	                         long_options, NULL)) != -1) {
 		switch (c) {
-        case 'b':
-            allow_bad_names = true;
-            break;
+			case 'b':
```

Comment:
Extra tab here?

Reply:
@tpimh, you're correct. I've pushed another update.